### PR TITLE
smth tests

### PR DESCRIPTION
**Pull request by glcanvas**

### CODE REVIEW

File: task2/boundaryCase.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+a=" hello"
+b="\'"
```
Comment (pva701) [inline task2/boundaryCase.sh]: this is not a valid structure in our mini shell

File: task2/standartTests.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+a=\"
```
Comment (pva701) [inline task2/standartTests.sh:1]: this is not a valid structure in our mini shell

File: task3/commandsTest2.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+c="1"
+b="2"
+echo " hello"
+
+
+
+a=\"
```
Comment (pva701) [inline task3/commandsTest2.sh:7]: it's not a valid structure

**Review by pva701 — CHANGES_REQUESTED** [review]

File: task2/standartTests.sh (hunk 2 above)
Comment (glcanvas) [reply task2/standartTests.sh:1]: доне

**Pull request merged**